sections/03_categorize.md

```diff
@@ -196,55 +196,73 @@ this field.
-addressed by other sections:  1) Data (and sometimes learnt
-features/representations)  need to be validated by clinicial experts.   2) The
-amount of expert labels needs to be sufficient large.
+Lack of true labels is perhaps among the biggest obstacles for EHR-based
+analyses that employ machine learning (e.g., phenotyping). Popular deep
+learning (and machine learning) methods are often used to tackle classification
+tasks and thus require ground-truth labels for training.  For EHRs,
+unfortunately, this means that researchers need to hire multiple clinicians to
+manually read and annotate individual patients' records through a process called
+chart review. This allows researchers to assign "true" labels, i.e. those that
+match our best available knowledge. Depending on the application, sometimes the
+features constructed by algorithms also need to be manually validated and
+interpreted by clinicians. This can be time consuming and expensive
+[@doi:10.1016/j.ijmedinf.2016.09.014]. Because of these costs, much of this
+research, including the work cited in this review, skips the process of expert
+review. Clinicians' skepticism for research without expert review may greatly
+dampen their enthusiasm for the work and consequently reduce its impact. To
+date, even well-resourced large national consortia have been challenged by the
+task of acquiring enough expert-validated labeled data. For instance, in the
+eMERGE consortia and PheKB database [@url:https://phekb.org/implementations],
+most samples with expert validation contain only 100 to 300 patients. These
+datasets are quite small, even for simple machine learning algorithms. The
+challenge is greater for deep learning models with many parameters. While
+unsupervised and semi-supervised approaches can help with small sample sizes,
+the field would benefit greatly from large collections of anonymized records in
+which a substantial number of records have undergone expert review.
+
+This challenge is not unique to EHR-based studies. Work on medical images,
+-omics data in applications for which detailed metadata are required, and other
+applications for which labels are costly to obtain will be hampered as long as
+abundant curated data are unavailable. Unsupervised and semi-supervised methods
+provide one path forward [@doi:10.1016/j.jbi.2016.10.007], as do adversarial
+training examples [@doi:10.1101/095786]. We also expect the recently described
+data programming strategy [@arxiv:1605.07723] to play a role in addressing this
+challenge. In data programming, noisy automated labeling functions are
+integrated. While numerous writers have described data as the new oil [@url:http://ana.blogs.com/maestros/2006/11/data_is_the_new.html, @url:https://medium.com/twenty-one-hundred/data-is-the-new-oil-a-ludicrous-proposition-1d91bba4f294],
+Ratner, Bach, and Ré described training data as "The _New_ New Oil" [@url:http://hazyresearch.github.io/snorkel/blog/weak_supervision.html].
```

Comment:
The phrase "_New_ New Oil" and how it differs from "New Oil" does not make sense without reading the reference.  I suggest explaining that phrase.